config/src/main/java/com/typesafe/config/impl/ConfigImpl.java

```diff
@@ -252,26 +271,26 @@ static AbstractConfigValue fromAnyRef(Object object, ConfigOrigin origin,
                                 "bug in method caller: not valid to create ConfigObject from map with non-String key: "
                                         + key);
                     AbstractConfigValue value = fromAnyRef(entry.getValue(),
-                            origin, mapMode);
+                    		origin, origin, mapMode);
```

Comment:
it is sort of hard to tell on github but it looks like there may be a tab character for indentation here; should be only spaces